from_csv_to_json.py

```diff
@@ -0,0 +1,58 @@
+from converter_file_format import Converter
```

Comment:
Where is Converter class implementation? Code you provided doesn't work without it.